src/index.js

```diff
@@ -0,0 +1,39 @@
+import { DateTime } from 'luxon';
+
+import Library from './assets/library.js';
+// eslint-disable-next-line max-classes-per-file
+
+// eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] Kindly make sure you don't disable linter rules in your code as this goes against the best practice set by Microverse regarding the usage of linter checks. ✔